nebu/cli/plugin.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+import imp
+from pathlib import Path
+from importlib.util import spec_from_file_location, module_from_spec
+
+import click
+from lxml import etree
+from litezip import (
+    parse_collection,
+    parse_module
+)
+
+from ._common import common_params
+
+@click.command()
+@common_params
+@click.argument('plugin',
+                type=click.Path(exists=True, dir_okay=False))
+@click.argument('content_dir',
+                type=click.Path(exists=True, file_okay=False))
+@click.argument('plugin_args', nargs=-1)
+@click.pass_context
+def plugin(ctx, plugin, content_dir, plugin_args):
```

Comment:
I like the intention here. I'm not sure this is the way to put it into action. In my mind plugins are something that plug into an existing operation rather than be their own operation. But maybe that's just my misconception of things.

So in other words, shouldn't plugins run on for example the `get` operation? Or be defined in such a way that their name implies what operation they plug into?

pytest has a great plug-in system for instance. That'd be something to look into emulating.

Reply:
I definitely agree 'plugins' doesn't seem like the right word for what this is, since it doesn't do or intend to do anything to enhance or modify a particular function of neb. Do you have any suggestions for what to call this?

That being said, I think there is value in supporting these CM tasks with external scripts (that perhaps colby could maintain) while trying to maintain a level of quality/consistency and without adding bloat to neb. With that context in mind, what are some alternatives for how to accomplish that, and how does this compare to those?